packages/tasit-action/src/NFT.test.js

```diff
@@ -93,66 +100,94 @@ describe("TasitAction.NFT", () => {
 
   describe("ERC721 functions", () => {
     const tokenId = 1;
+    const zeroAddress = "0x0000000000000000000000000000000000000000";
 
-    beforeEach("should mint one token to owner", async () => {
-      const balanceBefore = await fullNFT.balanceOf(owner.address);
-      expect(toBN(balanceBefore)).to.be.bignumber.equal(toBN(0));
+    beforeEach("should mint one token to ana", async () => {
+      const balanceBefore = await fullNFT.balanceOf(ana.address);
+      expect(balanceBefore.toNumber()).to.equal(0);
 
-      txSubscription = fullNFT.mint(owner.address, tokenId);
-      await txSubscription.waitForNonceToUpdate();
+      contractSubscription = fullNFT.subscribe();
+
+      txSubscription = fullNFT.mint(ana.address, tokenId);
```

Comment:
`msg.sender` here is the `owner`, right?

Reply:
yes

Reply:
Okay, good to know. Thanks. Resolving this.